packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -262,7 +262,19 @@ mixin RendererBinding
   )
   late final PipelineOwner pipelineOwner = PipelineOwner(
```

Comment:
This is deprecated, I think the changes should be made in `createRootPipelineOwner` instead?